libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -166,6 +168,12 @@ namespace eosio { namespace chain {
       const chainbase::database& db;
    };
 
+#define _ADD_PAREN_1(...) ((__VA_ARGS__)) _ADD_PAREN_2
+#define _ADD_PAREN_2(...) ((__VA_ARGS__)) _ADD_PAREN_1
+#define _ADD_PAREN_1_END
+#define _ADD_PAREN_2_END
+#define _WRAPPED_SEQ(SEQ) BOOST_PP_CAT(_ADD_PAREN_1 SEQ, _END)
+
```

Comment:
These are reserved identifiers.

Reply:
not sure if @b1bart has any thoughts on why there are named that way

Reply:
bad habit I suppose.  feel free to rename them and properly `undefine` them it is merely a nod to the idea that they are "internal".